database/migrations/postgres/000004_add_inserted_at.up.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE transactions
+ADD COLUMN  inserted_at TIMESTAMP WITH TIME ZONE  DEFAULT CURRENT_TIMESTAMP NOT NULL;
```

Comment:
These background jobs will be running  frequently and with this approach we will be querying 2 tables instead of 1 that we could. I'm not sure it's best idea.

Reply:
can you please explain why adding another pretty standard column is less optimal than what you are suggesting?

Reply:
It's just an alternative. But we can try your approach first